src/govuk/components/details/details.test.js

```diff
@@ -5,130 +5,162 @@ const PORT = configPaths.ports.test
 
 const baseUrl = 'http://localhost:' + PORT
 
-describe('/components/details', () => {
-  describe('/components/details/preview', () => {
+describe('details', () => {
```

Comment:
Do you think it's worth adding a test that looks at the native details element (`/components/details/preview`) and asserts that it doesn't e.g. add all the aria attributes?

Reply:
Yeah this would check the feature detection works, good idea